core/src/main/java/com/expedia/www/haystack/client/idgenerators/TimeBasedUUIDGenerator.java

```diff
@@ -21,11 +21,15 @@
 
 import java.util.UUID;
 
-
 public class TimeBasedUUIDGenerator implements IdGenerator {
```

Comment:
I couldn't find documentation about this generator. I'm assuming this would be for high volume applications where we need a better random seed?

Reply:
This generator is faster in terms of performance. High volume applications can use this. Other Random generator is based on Java pesudo random generation that is kinda slow.